### ISSUE TITLE
Crash when initializing server with "magic" module disabled

### ISSUE THREAD

**Issue by Mimickal** [Railcraft/Railcraft#1305]
My forge server crashes on startup when attempting to load Railcraft with the config option `B:"railcraft:magic"=false` set in `modules.cfg`. Setting this option to `true` causes the server to load as expected.

Versions:
 - Minecraft 1.10.2
 - Forge 12.18.3.2281
 - Railcraft 10.1.2

latest.log output:
```
[23:26:17] [Server thread/INFO]: Starting minecraft server version 1.10.2
[23:26:17] [Server console handler/ERROR]: Exception handling console input
java.io.IOException: The handle is invalid
	at java.io.FileInputStream.readBytes(Native Method) ~[?:1.8.0_131]
	at java.io.FileInputStream.read(Unknown Source) ~[?:1.8.0_131]
	at java.io.BufferedInputStream.read1(Unknown Source) ~[?:1.8.0_131]
	at java.io.BufferedInputStream.read(Unknown Source) ~[?:1.8.0_131]
	at sun.nio.cs.StreamDecoder.readBytes(Unknown Source) ~[?:1.8.0_131]
	at sun.nio.cs.StreamDecoder.implRead(Unknown Source) ~[?:1.8.0_131]
	at sun.nio.cs.StreamDecoder.read(Unknown Source) ~[?:1.8.0_131]
	at java.io.InputStreamReader.read(Unknown Source) ~[?:1.8.0_131]
	at java.io.BufferedReader.fill(Unknown Source) ~[?:1.8.0_131]
	at java.io.BufferedReader.readLine(Unknown Source) ~[?:1.8.0_131]
	at java.io.BufferedReader.readLine(Unknown Source) ~[?:1.8.0_131]
	at net.minecraft.server.dedicated.DedicatedServer$2.run(DedicatedServer.java:101) [ld$2.class:?]
[23:26:58] [Server thread/INFO]: Loading properties
[23:26:58] [Server thread/INFO]: Default game type: CREATIVE
[23:26:58] [Server thread/INFO]: Generating keypair
[23:26:58] [Server thread/INFO]: Starting Minecraft server on *:35565
[23:26:58] [Server thread/INFO]: Using default channel type
[23:27:04] [Server thread/ERROR]: Encountered an unexpected exception
net.minecraftforge.fml.common.LoaderExceptionModCrash: Caught exception from Railcraft (railcraft)

Caused by: java.lang.RuntimeException: Items must be initialized in Construction or PreInit!
	at mods.railcraft.common.plugins.forge.RailcraftRegistry.register(RailcraftRegistry.java:132) ~[RailcraftRegistry.class:?]
	at mods.railcraft.common.items.RailcraftItems.register(RailcraftItems.java:167) ~[RailcraftItems.class:?]
	at mods.railcraft.common.items.RailcraftItems.getStack(RailcraftItems.java:222) ~[RailcraftItems.class:?]
	at mods.railcraft.common.core.IRailcraftObjectContainer.getStack(IRailcraftObjectContainer.java:54) ~[IRailcraftObjectContainer.class:?]
	at mods.railcraft.common.core.IRailcraftObjectContainer.getStack(IRailcraftObjectContainer.java:49) ~[IRailcraftObjectContainer.class:?]
	at mods.railcraft.common.modules.ModuleFactory$1.postInit(ModuleFactory.java:414) ~[ModuleFactory$1.class:?]
	at mods.railcraft.common.modules.RailcraftModulePayload$BaseModuleEventHandler.postInit(RailcraftModulePayload.java:82) ~[RailcraftModulePayload$BaseModuleEventHandler.class:?]
	at mods.railcraft.common.modules.RailcraftModuleManager$Stage$4.passToModule(RailcraftModuleManager.java:285) ~[RailcraftModuleManager$Stage$4.class:?]
	at mods.railcraft.common.modules.RailcraftModuleManager.processStage(RailcraftModuleManager.java:224) ~[RailcraftModuleManager.class:?]
	at mods.railcraft.common.modules.RailcraftModuleManager.postInit(RailcraftModuleManager.java:211) ~[RailcraftModuleManager.class:?]
	at mods.railcraft.common.core.Railcraft.postInit(Railcraft.java:198) ~[Railcraft.class:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_131]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_131]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_131]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_131]
	at net.minecraftforge.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:616) ~[forge-1.10.2-12.18.3.2281-universal.jar:?]
	at sun.reflect.GeneratedMethodAccessor11.invoke(Unknown Source) ~[?:?]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_131]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_131]
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74) ~[minecraft_server.1.10.2.jar:?]
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47) ~[minecraft_server.1.10.2.jar:?]
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322) ~[minecraft_server.1.10.2.jar:?]
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304) ~[minecraft_server.1.10.2.jar:?]
	at com.google.common.eventbus.EventBus.post(EventBus.java:275) ~[minecraft_server.1.10.2.jar:?]
	at net.minecraftforge.fml.common.LoadController.sendEventToModContainer(LoadController.java:243) ~[LoadController.class:?]
	at net.minecraftforge.fml.common.LoadController.propogateStateMessage(LoadController.java:221) ~[LoadController.class:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_131]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_131]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_131]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_131]
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74) ~[minecraft_server.1.10.2.jar:?]
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47) ~[minecraft_server.1.10.2.jar:?]
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322) ~[minecraft_server.1.10.2.jar:?]
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304) ~[minecraft_server.1.10.2.jar:?]
	at com.google.common.eventbus.EventBus.post(EventBus.java:275) ~[minecraft_server.1.10.2.jar:?]
	at net.minecraftforge.fml.common.LoadController.distributeStateMessage(LoadController.java:145) ~[LoadController.class:?]
	at net.minecraftforge.fml.common.Loader.initializeMods(Loader.java:810) ~[Loader.class:?]
	at net.minecraftforge.fml.server.FMLServerHandler.finishServerLoading(FMLServerHandler.java:108) ~[FMLServerHandler.class:?]
	at net.minecraftforge.fml.common.FMLCommonHandler.onServerStarted(FMLCommonHandler.java:333) ~[FMLCommonHandler.class:?]
	at net.minecraft.server.dedicated.DedicatedServer.func_71197_b(DedicatedServer.java:214) ~[ld.class:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:431) [MinecraftServer.class:?]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_131]
[23:27:04] [Server thread/ERROR]: This crash report has been saved to: C:\Users\Donut\AppData\Roaming\.minecraft\.\crash-reports\crash-2017-08-03_23.27.04-server.txt
[23:27:04] [Server thread/INFO]: Stopping server
[23:27:04] [Server thread/INFO]: Saving worlds
[23:27:04] [Server Shutdown Thread/INFO]: Stopping server
```

**Comment by liach** [comment]
Thanks for the report!

**Comment by liach** [comment]
Should be fixed by #1177 already. Cannot reproduce in latest alpha.
